core/src/main/java/org/frankframework/logging/IbisPatternLayout.java

```diff
@@ -99,9 +99,7 @@ public String toSerializable(final LogEvent event) {
 
 		@Override
 		public StringBuilder toSerializable(final LogEvent event, final StringBuilder buffer) {
-			final int len = formatters.length;
-			for (int i = 0; i < len; i++) {
-				PatternFormatter formatter = formatters[i];
+			for (PatternFormatter formatter : formatters) {
```

Comment:
Kan ook met een lambda:

    Stream.of(formatters).forEach(formatter -> formatter.format(event, buffer));